package.json

```diff
@@ -27,25 +28,34 @@
     "test": "npm run build && npm run build:babel-test && wct",
     "checksize": "rollup -c ; rm lit-element.bundled.js",
     "format": "find src test | grep '\\.js$\\|\\.ts$' | xargs clang-format --style=file -i",
-    "lint": "tslint --project ./",
+    "lint": "npm run lint:eslint",
+    "lint:eslint": "eslint 'src/**/*.{js,ts}'",
```

Comment:
Why two separate scripts here?

Reply:
Nothing really special. In other projects I can do more lint (CSS, LitAnalayzer, etc)

Reply:
Updated!